perses/rjmc/topology_proposal.py

```diff
@@ -1,9 +1,17 @@
 """
 This file contains the base classes for topology proposals
 """
+### implement updates to TopologyProposal
+### calculate logp in different ways in different subclasses
 
 import simtk.openmm.app as app
 from collections import namedtuple
+<<<<<<< HEAD
```

Comment:
I thought I had found all of these and removed them but clearly missed a couple, thanks for the catch!